docs/reference/migration/migrate_8_0/http.asciidoc

```diff
@@ -0,0 +1,9 @@
+[float]
+[[breaking_80_http_changes]]
+=== HTTP changes
+
+[float]
+==== Removal of old HTTP settings
+
+The `http.tcp_no_delay` setting was deprecated in 7.x and has been removed in 8.0. It has been replaced by
+`http.tcp.no_delay`.
```

Comment:
We typically add a "deprecated[7.1.0]" to the setting's description in the release where it's deprecated, per https://github.com/elastic/docs#additions-and-deprecations
I expected to find http.tcp_no_delay defined in https://www.elastic.co/guide/en/elasticsearch/reference/7.x/modules-http.html or https://www.elastic.co/guide/en/elasticsearch/reference/7.x/modules-network.html 
@esdocs Is there somewhere else this setting is defined?

Reply:
The documentation for these settings was updated to refer to the new settings in 6.7 (including 7.0, 7.x, and 8.0). The commit to officially deprecate them was added to 7.1. What approach should I take in this scenario?

As for the http socket settings, the socket settings appear to be inconsistently documented. They are documented on the network page and for profiles on the transport page. But not on the http page or for the non-profile based transport settings. 

I guess this is a case that we need to improve?

Reply:
I had a quick chat with the @es-docs folks and they said from a support perspective, documenting that they’re no longer available makes sense, even if we never officially documented them in the first place. And it seems like including them in “breaking changes” for release X (where they were deprecated) and X+1 (where they were discontinued) is appropriate.